### DISCUSSION TITLE
Just upgraded...

### DISCUSSION THREAD

**Discussion by adm2k** [netalertx/NetAlertX#743] [Q&A]
Just upgraded to the latest version and am single this, what is the icon on the right representing? When I click on it it displays the same page as the other (My devices).

![image](https://github.com/user-attachments/assets/018f2e2e-f723-4b7e-9702-013a9e9e21df)

**Comment by jokob-sk** [comment]
I think that's "All devices". My devices is a subset of all devices (based on the `UI_MY_DEVICES` setting). It's possible you need to refresh the cache for the label to show up.  You can also pick which tiles are shown with the `UI_shown_cards` setting.

**Comment by adm2k** [comment]
Thanks, clearing cache resolved the issue. I had done that earlier - or at least I thought I did :)
